RHIDP-2264 : add extra app-config configuration for AKS Operator installation procedure (#217)
redhat-developer/red-hat-developers-documentation-rhdh: modules/admin/proc-rhdh-deploy-aks-using-operator.adoc

----
--

> . Create a ConfigMap named `app-config-rhdh` containing the {product-short} configuration using the following template:

Reviewer: we can call this an example rather than specifying it as a template. because templates have different meaning in different contexts. I see a completely different section for templates here: https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-217/admin-rhdh/#assembly-admin-templates.

better to avoid any confusion.

```suggestion
. Create a ConfigMap named `app-config-rhdh` containing the {product-short} configuration as shown in the following example:
```

Reviewer: Alternate suggestion: Create a ConfigMap named `app-config-rhdh` containing the {product-short} configuration by using the following YAML code:

Reviewer 2: used example. Done

+
--
[source,yaml]
----
apiVersion: v1
kind: ConfigMap
metadata:
  name: app-config-rhdh
data:
  "app-config-rhdh.yaml": |
    app:
      title: Red Hat Developer Hub

>       baseUrl: https://<rhdh_dns_name>

Reviewer: Suggestion: Maybe we can add a line or two that explains what does rhdh_dns_name denote and from where the user can get this name?

Reviewer 2: It's actually the app_address. I'm changing rhdh_dns_name to app_address.

    backend:
      auth:
        keys:
          - secret: "${BACKEND_SECRET}"
      baseUrl: https://<rhdh_dns_name>
      cors:
        origin: https://<rhdh_dns_name>
----
--

> . Create a Secret named `secrets-rhdh` and add a key named `BACKEND_SECRET` with a `Base64-encoded` string as value:

Reviewer: ```suggestion
. Create a Secret named `secrets-rhdh` and add a key named `BACKEND_SECRET` with `Base64-encoded` string value as shown in the following example:
```

Reviewer 2: Done

+
--
[source,yaml]
----
apiVersion: v1
kind: Secret
metadata:
  name: secrets-rhdh
stringData:

>   # TODO: See https://backstage.io/docs/auth/service-to-service-auth/#setup

Reviewer: do we really need this piece of line?? if not, we can remove it.

Reviewer 2: removed
